app/javascript/gobierto_dashboards/webapp/containers/contracts/Summary.vue

```diff
@@ -169,13 +188,26 @@ export default {
   created() {
     this.items = this.buildItems();
     this.columns = assigneesColumns;
+
+    EventBus.$on('filtered-items-grouped', (data, value) => this.updateItems(data, value))
   },
   beforeDestroy(){
     EventBus.$off('refresh-summary-data');
   },
   methods: {
-    refreshSummaryData(){
-      this.contractsData = this.$root.$data.contractsData;
+    updateItems(data, value) {
+      this.value = value
+      this.contractsData = data
+      this.items = this.buildItems();
+
+      EventBus.$emit('send-filtered-items', this.items)
```

Comment:
Chaining events. Why so? This comes from `filtered-items-grouped` then it does stuff, and does it send a new event `send-filtered-items`? Too twisted.